drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -598,8 +527,13 @@ static bool nrf5_tx_at(struct nrf5_802154_data *nrf5_radio, struct net_pkt *pkt,
 		.extra_cca_attempts = max_extra_cca_attempts,
 #endif
 	};
-	uint64_t tx_at = target_time_convert_to_64_bits(net_ptp_time_to_ns(net_pkt_timestamp(pkt)));
 
+	/* The timestamp points to the start of PHR but `nrf_802154_transmit_raw_at`
+	 * expects a timestamp pointing to start of SHR.
+	 */
+	uint64_t tx_at = net_ptp_time_to_ns(net_pkt_timestamp(pkt)) / NSEC_PER_USEC;
+
+	tx_at -= NRF5_SHR_DURATION_US;
```

Comment:
No need to add this symbol. I think [this function](https://github.com/zephyrproject-rtos/hal_nordic/blob/master/drivers/nrf_802154/driver/include/nrf_802154.h#L451) can be used instead.

Reply:
Doh! Of course - sorry, that was some leftover from prior iterations. Fixed.